src/bug-filing/azure-boards/azure-boards-bug-filing-service.tsx

```diff
@@ -1,52 +1,52 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 import { isEmpty } from 'lodash';
-import * as React from 'react';
-
+import { IDropdownOption } from 'office-ui-fabric-react/lib/Dropdown';
 import { EnvironmentInfo } from '../../common/environment-info-provider';
-import { NamedSFC } from '../../common/react/named-sfc';
 import { CreateIssueDetailsTextData } from '../../common/types/create-issue-details-text-data';
 import { BugServicePropertiesMap } from '../../common/types/store-data/user-configuration-store';
 import { BugFilingService } from '../types/bug-filing-service';
-import { SettingsFormProps } from '../types/settings-form-props';
+import { AzureBoardsSettingsForm } from './azure-boards-settings-form';
+
+const AzureDevOpsBugFilingServiceKey = 'azureBoards';
 
-const AzureDevOpsBugFilingServiceKey = 'azureDevOps';
+export type AzureBoardsIssueDetailField = 'reproSteps' | 'description';
 
-export type AzureDevOpsBugFilingSettings = {
+export interface AzureBoardsIssueDetailLocationDropdownOption extends IDropdownOption {
+    key: AzureBoardsIssueDetailField;
+}
+
+export type AzureBoardsBugFilingSettings = {
     projectURL: string;
-    issueDetailsLocationField: string;
+    issueDetailsField: AzureBoardsIssueDetailField;
 };
 
-function buildStoreData(projectURL: string, issueDetailsLocationField: string): AzureDevOpsBugFilingSettings {
+function buildStoreData(projectURL: string, issueDetailsField: AzureBoardsIssueDetailField): AzureBoardsBugFilingSettings {
```

Comment:
`buildStoreData` seems to only be used in tests, for now at least... curious who is going to call it generically while knowing the right order for parameters?

Reply:
We've been building this piece by piece. Wiring up stories should use this in the near future. If we get to a point when we don't need this, we'll clean it up.